Find.md

```diff
@@ -0,0 +1,202 @@
+# JULEP find
+
+- **Title:** Reorganize Search and Find API
+- **Authors:** Milan Bouchet-Valat <<nalimilan@club.fr>>
+- **Created:** December 10, 2016
+- **Status:** work in progress
+
+## Abstract
+
+The current `find` and `search` families of functions are not very consistent as regards
```

Comment:
I always wonder which one is best. There's also "in regard to", but "with regard to" is clearly the most frequent (though quite below "regarding").